crates/turborepo-lib/src/run/cache.rs

```diff
@@ -360,6 +364,21 @@ impl TaskCache {
             globwalk::WalkType::All,
         )?;
 
+        debug!("files to cache: {:?}", files_to_be_cached.len());
+        let mut status = OutputSaveResult::Saved;
+        // Compare to 1 because the log file is always cached.
+        if files_to_be_cached.len() == 1 {
+            // Were we expecting to cache more than just the log file?
+            if validated_inclusions.len() > 1 {
+                status = OutputSaveResult::SavedEmpty;
+                if self.run_cache.skip_empty_cache {
+                    return Ok(OutputSaveResult::SkippedEmpty);
+                }
+            } else {
+                debug!("caching log output only for {:?}", self.task_id);
```

Comment:
Super minor, but I think this `debug` log is made unnecessary by the `debug` log earlier.